src/Components/Slides/SlidesForm.js

```diff
@@ -1,32 +1,107 @@
-import React, { useState } from 'react';
-import '../FormStyles.css';
+import axios from "axios";
+import React, { useState } from "react";
+import ClassicEditor from '@ckeditor/ckeditor5-build-classic';
+import { CKEditor } from '@ckeditor/ckeditor5-react';
+import "../FormStyles.css";
+import Swal from 'sweetalert2'
 
-const SlidesForm = () => {
-    const [initialValues, setInitialValues] = useState({
-        name: '',
-        description: ''
-    });
-
-    const handleChange = (e) => {
-        if(e.target.name === 'name'){
-            setInitialValues({...initialValues, name: e.target.value})
-        } if(e.target.name === 'description'){
-            setInitialValues({...initialValues, description: e.target.value})
+const SlidesForm = ({slides}) => {
+  const [initialValues, setInitialValues] = useState({
+    name: slides?.name ?? '',
+    description:slides?.description ?? '',
+    image:slides?.image ?? '',
+    order:slides?.order ?? '',
+  });
+  const hasSlides = !!slides
+  const handleSubmit = (e) => {
+    e.preventDefault();
+  };
+  const handleData = () => {
+    if(!hasSlides){
+    axios({
+      method: "POST",
+      url: "http://ongapi.alkemy.org/api/slides",
+      data: {
+        name: initialValues.name,
+        description: initialValues.description,
+        image: initialValues.image,
+      },
+    })
+      .then((res) => {
+        if (initialValues.name.length>4) {
+          Swal.fire(
+            'Slide Creado!'
+          )
+        } else {
+          Swal.fire({
+            title: 'Error!',
+            text: 'No se puede tener menos de 4 caracteristicas',
+            icon: 'error',
+            confirmButtonText: 'Ok, voy a editarlo!'
+          })
         }
-    }
-
-    const handleSubmit = (e) =>{
-        e.preventDefault();
-        console.log(initialValues);
-    }
+      })
+      .catch((err) => (err));
+    } else { axios({
```

Comment:
same of post, this should be on a SlideService and then import the function here. Also if its a post or a put is a business logic so it should be another function in the service,

Reply:
Done, i alredy changed it